addons/mail_group/models/mail_group.py

```diff
@@ -459,7 +459,7 @@ def _notify_members(self, message):
                 }
                 footer = self.env['ir.qweb']._render('mail_group.mail_group_footer', template_values, minimal_qcontext=True)
                 member_body = tools.append_content_to_html(body, footer, plaintext=False)
-
+                member_body = self.env['mail.render.mixin']._replace_local_links(member_body)
```

Comment:
As we already apply it on the body earlier wouldn't it make sense to only apply it on the footer here?